src/main/resources/Dockerfile

```diff
@@ -1,17 +1,11 @@
-ARG TARANTOOL_VERSION=2.11.2-centos7
+ARG TARANTOOL_VERSION=2.11.2-ubuntu20.04
 FROM tarantool/tarantool:${TARANTOOL_VERSION} AS cartridge-base
 
-# system preparations because docker mount directory as a root
-ARG TARANTOOL_SERVER_USER="root"
-ARG TARANTOOL_SERVER_GROUP="root"
-USER $TARANTOOL_SERVER_USER:$TARANTOOL_SERVER_GROUP
-RUN groupadd $TARANTOOL_SERVER_GROUP && useradd -m -s /bin/bash $TARANTOOL_SERVER_USER || true
-
 # install dependencies
 # a yum bug requires setting ulimit, see https://bugzilla.redhat.com/show_bug.cgi?id=1537564
-RUN ulimit -n 1024 && \
```

Comment:
I would leave ulimit as it was

Reply:
You mean single command 2 strings?

Reply:
I meant you delete ulimit command at all

Reply:
Oh, it's a warkaround for a yum bug. Now I'm using apt-get

Reply:
for history: You've fixed it